src/components/node-list/node-list.test.js

```diff
@@ -142,6 +143,64 @@ describe('NodeList', () => {
     });
   });
 
+  describe('Pretty names in node list', () => {
+    const elements = (wrapper) =>
+      wrapper
+        .find('.MuiTreeItem-label')
+        .find('.pipeline-nodelist__row')
+        .map((row) => [
+          row.prop('title'),
+          !row.hasClass('pipeline-nodelist__row--disabled'),
+        ]);
+
+    it('shows full node names when pretty name is turned off', () => {
+      const wrapper = setup.mount(<NodeList />, {
+        beforeLayoutActions: [() => togglePrettyName(false)],
+      });
+      expect(elements(wrapper)).toEqual([
+        ['nested', true],
+        ['pipeline1', true],
+        ['pipeline2', true],
+        ['salmon', true],
+        ['bear', true],
+        ['bull', true],
+        ['cat', true],
+        ['dog', true],
+        ['elephant', true],
+        ['giraffe', true],
+        ['horse', true],
+        ['pig', true],
+        ['sheep', true],
+        ['parameters', false],
+        ['params:pipeline100.data_science.plankton', false],
+        ['params:rabbit', false],
```

Comment:
This set of tests, while does test the displayed title of the wrapper, is more structured to test the `enabled` feature as set out in the other tests - we won't need to test whether they are disabled but only need to test the names, so you can take out the check for the `disabled` class here. This will also apply to the test below from lines 184 - 199 :)

Reply:
Now, i get it . i was wondering why there's true and false when i copy/pasted it from another tests. thanks I have removed it :D